handlers/Services/index.js

```diff
@@ -9,7 +9,7 @@ import Icon from 'Icon';
 import Typography from 'Typography';
 
 const SERVICES = [
-  ['IOT', 'internet-of-things', 'Through interviews, joint ideation, storyboards and estimation, we give you an informed, fresh perspective to help you prioritize concepts for further investment.'],
+  ['Internet Of Things', 'internet-of-things', 'Through interviews, joint ideation, storyboards and estimation, we give you an informed, fresh perspective to help you prioritize concepts for further investment.'],
```

Comment:
The first key was also used to lookup the icon and a few other things I believe, so this may require a few other changes. I’ll pull down the code and run it later today to confirm.